tests/Avalonia.PlatformSupport.UnitTests/AssetLoaderTests.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Reflection;
+using Avalonia.PlatformSupport.Internal;
+using Moq;
+using Xunit;
+
+namespace Avalonia.PlatformSupport.UnitTests;
+
+public class AssetLoaderTests
+{
+    public class MockAssembly : Assembly {}
+
+    private const string AssemblyNameWithWhitespace = "Awesome Library";
+
+    private const string AssemblyNameWithNonAscii = "Какое-то-название";
+
+    static AssetLoaderTests()
+    {
+        var resolver = Mock.Of<AssemblyDescriptorResolver>();
```

Comment:
I'm seeing this error in these tests running locally:

```
System.ArgumentException: Type to mock must be an interface, a delegate, or a non-sealed, non-static class.
```

It doesn't seem to be failing CI because it looks like Avalonia.PlatformSupport.UnitTests isn't run on CI due to it not being added to the [nuke script](https://github.com/AvaloniaUI/Avalonia/blob/master/nukebuild/Build.cs#L217-L223).

Could you fix this @rstm-sf ?

Reply:
Sorry, it's fixed https://github.com/AvaloniaUI/Avalonia/pull/7989